community/graph-algo/src/main/java/org/neo4j/graphalgo/GraphAlgoFactory.java

```diff
@@ -375,11 +375,11 @@
     /**
      * See {@link #dijkstra(PathExpander, CostEvaluator)} for documentation
      *
-     * Instead of finding all shortest paths with equal cost, find the top {@param numberOfWantedPaths} paths.
+     * Instead of finding all shortest paths with equal cost, find the top {@code numberOfWantedPaths} paths.
      * This is usually slower than finding all shortest paths with equal cost.
      *
      * Uses a cost evaluator which uses the supplied property key to
-     * represent the cost (values of type <bold>double</bold>).
+     * represent the cost (values of type <b>double</b>).
```

Comment:
while this change is correct from an html point of view, I wonder if we might not actually want to format this as code...